app/src/main/java/com/github/se/wanderpals/ui/screens/notification/CreateNotification.kt

```diff
@@ -0,0 +1,140 @@
+  var titleError by remember { mutableStateOf(false) }
+  var descriptionError by remember { mutableStateOf(false) }
+
+  Surface(modifier = Modifier.padding(12.dp)) {
+    Column(
+        modifier = Modifier.fillMaxSize().padding(horizontal = 20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+      OutlinedButton(
+          modifier = Modifier.align(Alignment.Start).testTag("tripNotifGoBackButton"),
+          onClick = { onCancel() }) {
+            Icon(
+                imageVector = Icons.AutoMirrored.Rounded.ArrowBack,
+                contentDescription = "Back",
+            )
+          }
+    }
+
+    Column(
+        modifier = Modifier.fillMaxSize().padding(horizontal = 20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center) {
+          OutlinedTextField(
+              value = title,
+              onValueChange = { if (it.length <= MAX_NOTIF_TITLE_LENGTH) title = it },
+              label = { Text("Trip Notification Title*") },
+              modifier = Modifier.fillMaxWidth().testTag("inputNotificationTitle"),
+              isError = titleError,
+              singleLine = true)
+          Text(
+              text = "${title.length}/$MAX_NOTIF_TITLE_LENGTH",
+              modifier =
+                  Modifier.align(Alignment.End).padding(end = 8.dp).testTag("notifTitleLengthText"),
+              style = TextStyle(fontSize = 12.sp, color = Color.Gray))
+
+          Spacer(modifier = Modifier.height(12.dp))
+
+          OutlinedTextField(
+              value = description,
+              onValueChange = { description = it },
+              label = { Text("Trip Notification Description*") },
+              modifier =
+                  Modifier.fillMaxWidth().height(150.dp).testTag("inputNotificationDescription"),
+              isError = descriptionError,
+              singleLine = false)
+
+          Spacer(modifier = Modifier.height(24.dp))
+
+          Button(
+              modifier = Modifier.fillMaxWidth().height(50.dp).testTag("createNotificationButton"),
+              onClick = {
+                titleError =
+                    title.trim().isEmpty() // add trim() to remove leading and trailing whitespaces
+                descriptionError = description.trim().isEmpty()
+
+                if (!titleError && !descriptionError) {
+                  // Logic to create notification
+                  val tripNotification =
+                      TripNotification(
+                          notificationId = "", // modified by database
+                          userId = "", // modified by database
+                          title = title,
+                          userName = "", // modified by database
+                          description = description,
+                          timestamp = LocalDateTime.now())
+                  // if successful, call onSuccess(), otherwise onFailure()

```

Comment:
For the moment, it is not necessary to define onFailure() and onSuccess() case behaviour, for the moment we assume that after adding the notification in the view Model, we will just navigate back to the notification screen. 
We can assume that if the add of the notif fails, we also navigate back to the Notification screen. (We can change the behaviour lated if required.

Reply:
Okay! And if required, we can still modify it later :D